cosmo_tester/config_schemas/platform_openstack.yaml

```diff
@@ -3,8 +3,8 @@ plugin_package_name:
   description: Package name of plugin for this platform. This plugin must be installed on the infrastructure manager with visibility=global.
   default: cloudify-openstack-plugin
 plugin_version:
-  description: The expected version of this plugin.
-  default: 3.2.10
+  description: The expected version of this plugin (can be a regex).
+  default: 3\.\d\.\d
```

Comment:
I think this should be 3\.\d+\.\d+?

Reply:
^3\.\d+\.\d+$, escaping the dots, of course